config/routes.rb

```diff
@@ -247,8 +247,9 @@
     put '/user_authorization_confirmation/reset' => 'users/authorization_confirmation#update', as: :reset_user_authorization
     get '/sign_up/cancel/' => 'sign_up/cancellations#new', as: :sign_up_cancel
 
-    get '/return_to_sp/cancel' => 'return_to_sp#cancel'
-    get '/return_to_sp/failure_to_proof' => 'return_to_sp#failure_to_proof'
```

Comment:
for the 15 minute deploy window, should we preserve these old URLs and clean up in a future PR? Or do we expect that they're low enough traffic it won't be a big deal if they break?

Reply:
> for the 15 minute deploy window, should we preserve these old URLs and clean up in a future PR? 

Yeah, that sounds reasonable enough 👍

Reply:
Restored in 3ea3910.

Reply:
Follow-up removal at #5679